pkg/controller/common/unmanaged.go

```diff
@@ -0,0 +1,58 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package common
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	// ManagedAnnotation annotation
+	LegacyPauseAnnoation = "common.k8s.elastic.co/pause"
+	ManagedAnnotation    = "eck.k8s.elastic.co/managed"
+)
+
+var (
+
+	// CheckManagedRequeue is the default requeue to re-check if a resource is still unmanaged.
+	CheckManagedRequeue = reconcile.Result{Requeue: true, RequeueAfter: 10 * time.Second}
```

Comment:
I don't think we need that since we already watch the resource?

Reply:
True, I did not question the necessity of this and just carried it over from the existing code. Good thinking!